hubmapinventory/inventory.py

```diff
@@ -26,29 +27,31 @@
 
 
 ###############################################################################################################
-def __pprint(msg):
+def __pprint(msg: str):
     row = len(msg)
     h = "".join(["+"] + ["-" * row] + ["+"])
     result = "\n" + h + "\n" "|" + msg + "|" "\n" + h
     print(result)
 
 
-def __update_dataframe(dataset, temp, key):
+def __update_dataframe(
+    dataset: pd.DataFrame, temp: pd.DataFrame, key: str
+) -> pd.DataFrame:
     for index, datum in temp.iterrows():
         dataset.loc[index, key] = temp.loc[index, key]
     return dataset
 
 
 ###############################################################################################################
 def create(
-    hubmap_id,
-    token=None,
-    ncores=2,
-    compute_uuids=False,
-    dbgap_study_id=None,
-    recompute_file_extension=False,
-    debug=False,
-):
+    hubmap_id: str,
+    dbgap_study_id: str | None,
```

Comment:
I believe the union operator was added in 3.10 and I wonder if that might cause some conflicts when run from Airflow's 3.9 environment (this can be another entry in my saga of Python version struggles). I can let you know once I get it cooperating.